pkg/server/grpc/tracee.go

```diff
@@ -138,10 +588,26 @@ func convertDefinitionToProto(d events.Definition) *pb.EventDefinition {
 	}
 }
 
+func getGlobalID(e trace.Event) pb.Id {
+	// If syscall, need a translation. Use map table of internal syscall ID to external ID
+	if events.Core.GetDefinitionByID(events.ID(e.EventID)).IsSyscall() {
+		if idExternal, ok := syscallTranslationMap[events.ID(e.EventID)]; ok {
+			return pb.Id(idExternal)
+		}
+	} else { // If not syscall, access enum directly using event ID
+		// Return external ID just if found
+		if _, ok := pb.Id_name[int32(e.EventID)]; ok {
+			return pb.Id(e.EventID)
+		}
+	}
+	return pb.Id_unspecified
```

Comment:
Initializing an array at start, as suggested in a previous comment, will make this translation a simple access to the array at EventID index, which is better considering the fact this will be called for every event